sphinx_js/doclets.py

```diff
@@ -140,6 +143,39 @@ def analyzer_for(language):
         raise SphinxError('Unsupported value of js_language in config: %s' % language)
 
 
+def process_see(doclets):
+    """Process @see tags for all doclets."""
+    for doclet in doclets:
+        if 'see' in doclet:
+            see = doclet['see']
+            process_see_list(see)
+
+
+def process_see_list(see):
+    """Process a list of @see tag values.
+
+    Leaves any value with a space as freeform text.
+    Translates other values into RST crossreferences.
+    """
```

Comment:
Most other places in the code, we've been leaving an extra blank line at the end of multiline docstrings. Btw, I loooooove your good docstrings. So many people neglect them, but they are the first place I go to understand new code.